.eslintrc.js

```diff
@@ -35,6 +35,8 @@ module.exports = {
       // enable this rule to see literally everything missing jsdocs, this rule needs some refinement but is good as a sanity check.
       // 'jsdoc/require-jsdoc': [ERROR, {contexts:['TSInterfaceDeclaration TSPropertySignature', 'TSInterfaceDeclaration TSMethodSignature']}],
       'jsdoc/require-description': [ERROR, {exemptedBy: ['deprecated'], checkConstructors: false}],
+      'no-redeclare': OFF,
```

Comment:
If we turn this off, then it wouldn't catch something like this right?
```
// this is all one block
let someVar = 'foo';
...
let someVar = 5;
```

Reply:
o, i think i see, it'll still fail this because of the typescript-eslint/no-redeclare?

Reply:
Correct, we're turning off `no-redeclare` and turning on `typescript-eslint/no-redeclare`, which would take care of use-cases like that, plus allows function overloading.